### ISSUE TITLE
High/Low Tides Visual Chart (for regions with relevant data)

### ISSUE THREAD

**Issue by EnigmaSolved** [breezy-weather/breezy-weather#325]
### Describe your idea

Would love to see the addition of Tide data (when the relevant data is available in a given locale), a la the following examples:

https://www.tideguide.app/

https://www.tideschart.com/United-States/Virginia/City-of-Hampton/Buckroe-Beach/

Both the current day and multi-day Tide forecast would be great.

:)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] My idea is not related to an existing feature described in the [Homepage](https://github.com/breezy-weather/breezy-weather/blob/main/docs/HOMEPAGE.md) document.
- [X] I have updated the app to **[latest version](https://github.com/breezy-weather/breezy-weather/releases/latest)**.

**Comment by papjul** [comment]
This is a very specific request, as not something we usually see in weather apps, so I suggest keep using a dedicated app for now, as it will not be treated in priority (if ever implemented one day given the size of the backlog).

Linking to documentation of endpoints I found for our currently supported weather sources:
https://open-meteo.com/en/docs/marine-weather-api
https://apidev.accuweather.com/developers/TidalForecastAPIGuide
https://api.met.no/weatherapi/tidalwater/1.1/documentation

All other sources don’t seem to support it.

**Comment by papjul** [comment]
I leave it open to contributions, but please contact us before starting anything as it is not a trivial task.
